<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>firstPage</class>
 <widget class="QWidget" name="firstPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1850</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>1411</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="topLayout">
    <item>
     <widget class="QCheckBox" name="checkBox_figuresLaserDot">
      <property name="text">
       <string>show the white Dot where the figure will be centered</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_refreshBackground">
      <property name="text">
       <string>Refresh Background</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Line</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="lineTable">
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>l =</string>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string>phi =</string>
           </property>
          </item>
          <item row="2" column="0">
           <property name="text">
            <string>theta = </string>
           </property>
          </item>
          <item row="3" column="0">
           <property name="text">
            <string>repetitions =</string>
           </property>
          </item>
          <item row="4" column="0">
           <property name="text">
            <string>vel = </string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="line_cut">
          <property name="text">
           <string>line_cut</string>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_line_cut_at_dot">
          <property name="text">
           <string>line_cut at pos. of white dot</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Rectangle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="recTable">
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>a =</string>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string>b =</string>
           </property>
          </item>
          <item row="2" column="0">
           <property name="text">
            <string>phi0 =</string>
           </property>
          </item>
          <item row="3" column="0">
           <property name="text">
            <string>rotX =</string>
           </property>
          </item>
          <item row="4" column="0">
           <property name="text">
            <string>rotZ =</string>
           </property>
          </item>
          <item row="5" column="0">
           <property name="text">
            <string>vel. =</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rec_cut">
          <property name="text">
           <string>rec_cut</string>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_rec_cut_at_dot">
          <property name="text">
           <string>rec_cut at pos. of white dot</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Stripes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="stripesTable">
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>a =</string>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string>b =</string>
           </property>
          </item>
          <item row="2" column="0">
           <property name="text">
            <string>phi0 = </string>
           </property>
          </item>
          <item row="2" column="1">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="3" column="0">
           <property name="text">
            <string>rotX = </string>
           </property>
          </item>
          <item row="4" column="0">
           <property name="text">
            <string>rotZ = </string>
           </property>
          </item>
          <item row="5" column="0">
           <property name="text">
            <string>#lines between =</string>
           </property>
          </item>
          <item row="6" column="0">
           <property name="text">
            <string>vel. =</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stripes_cut">
          <property name="text">
           <string>stripes_cut</string>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_stripes_cut_at_dot">
          <property name="text">
           <string>stripes_cut at pos. of white dot</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Polygon</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="polyTable">
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>R =</string>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string>steps =</string>
           </property>
          </item>
          <item row="2" column="0">
           <property name="text">
            <string>phi0 =</string>
           </property>
          </item>
          <item row="3" column="0">
           <property name="text">
            <string>rotX = </string>
           </property>
          </item>
          <item row="4" column="0">
           <property name="text">
            <string>rotZ = </string>
           </property>
          </item>
          <item row="5" column="0">
           <property name="text">
            <string>vel. = </string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="poly_cut">
          <property name="text">
           <string>poly_cut</string>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_poly_cut_at_dot">
          <property name="text">
           <string>poly_cut at pos. of white dot</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Spiral</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="spiralTable">
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>R_start =</string>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string>R_end =</string>
           </property>
          </item>
          <item row="2" column="0">
           <property name="text">
            <string>steps = </string>
           </property>
          </item>
          <item row="3" column="0">
           <property name="text">
            <string>sectors = </string>
           </property>
          </item>
          <item row="4" column="0">
           <property name="text">
            <string>phi0 = </string>
           </property>
          </item>
          <item row="5" column="0">
           <property name="text">
            <string>rotX = </string>
           </property>
          </item>
          <item row="6" column="0">
           <property name="text">
            <string>rotZ =</string>
           </property>
          </item>
          <item row="7" column="0">
           <property name="text">
            <string>vel. = </string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="spiral_cut">
          <property name="text">
           <string>spiral_cut</string>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_spiral_cut_at_dot">
          <property name="text">
           <string>spiral_cut at pos. of white dot</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineTable</tabstop>
  <tabstop>line_cut</tabstop>
  <tabstop>recTable</tabstop>
  <tabstop>rec_cut</tabstop>
  <tabstop>stripesTable</tabstop>
  <tabstop>stripes_cut</tabstop>
  <tabstop>polyTable</tabstop>
  <tabstop>poly_cut</tabstop>
  <tabstop>spiralTable</tabstop>
  <tabstop>spiral_cut</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
